ledger-api/grpc-definitions/com/daml/ledger/api/v1/package_service.proto

```diff
@@ -9,6 +9,8 @@ option java_outer_classname = "PackageServiceOuterClass";
 option java_package = "com.daml.ledger.api.v1";
 option csharp_namespace = "Com.Daml.Ledger.Api.V1";
 
+// TODO this needs to be also multi-domain
```

Comment:
We propose to not expose party/configuration/package service endpoints with the assumptions:
* The original 2.0 LAPI + canton admin API is sufficient for CN
* If original 2.0 LAPI needs to be used, the 2.0 offsets are not disturbing
@moritzkiefer-da @meiersi-da WDYT?

(as going to 3.0 we probably anyway need to analyze usage patterns and craft a sufficient API)